### PR TITLE
Let wazuh-db IT to use registered agent ids

### LINKED ISSUE #1450: Adapt wazuh-db Integration Tests
Hi team!!

After the development of https://github.com/wazuh/wazuh/pull/8902/ we need to change the wazuh-db ITs in order to adapt them to the new behavior.

Now the test needs to use registered agents for the messages, otherwise, the test will fail. One option is to create some functions to mock agents and preventing failures in the tests.

### PR DESCRIPTION
**Pull request by antoniomanuelfr**
|Related issue|
|---|
|#1450|

Hi team
After the development of https://github.com/wazuh/wazuh/pull/8902, `wazuh-db` will not process queries from unregistered agents, this was causing failures in the ITs. This PR aims to fix these failures.

Closes #1450

- [x] Proven that tests **pass** when they have to pass.
- [x] Proven that tests **fail** when they have to fail.

- [x] Python codebase satisfies PEP-8 style style guide. `pycodestyle --max-line-length=120 --show-source --show-pep8 file.py`.
- [x] Python codebase is documented following the Google Style for Python docstrings.
- [x] The test is documented in wazuh-qa/docs.
- [ ] `provision_documentation.sh` generate the docs without errors.

### CODE REVIEW

File: tests/integration/test_wazuh_db/test_wazuh_db.py [hunk 1]
```diff
@@ -17,11 +17,19 @@
...
-messages_files = os.listdir(test_data_path)
-module_tests = list()
-for file in messages_files:
-    with open(os.path.join(test_data_path, file)) as f:
-        module_tests.append((yaml.safe_load(f), file.split("_")[0]))
+agent_message_files = os.path.join(test_data_path, 'agent')
+global_message_files = os.path.join(test_data_path, 'global')
+
+agent_module_tests = list()
+global_module_tests = list()
```
Comment (jmv74211) [inline tests/integration/test_wazuh_db/test_wazuh_db.py:24]:
```suggestion
agent_module_tests = []
global_module_tests = []
```

File: tests/integration/test_wazuh_db/test_wazuh_db.py [hunk 2]
```diff
@@ -56,76 +64,139 @@ def regex_match(regex, string):
     return re.match(regex, string)
 
 
+def execute_wazuh_db_query(command: str):
+    """Function to send a command to the wazuh-db socket.
+    Args:
+        command(str): Message to send to the socket.
+    Returns
```
Comment (jmv74211) [inline tests/integration/test_wazuh_db/test_wazuh_db.py:71]:
```suggestion
    Returns:
```

File: tests/integration/test_wazuh_db/test_wazuh_db.py [hunk 3]
```diff
@@ -56,76 +64,139 @@ def regex_match(regex, string):
...
+        A response from the socket
```
Comment (jmv74211) [inline tests/integration/test_wazuh_db/test_wazuh_db.py:72]: Please, follow this format `Type: Message`

**Review by Molter73 — APPROVED** [review]
LGTM!

**Comment by antoniomanuelfr** [comment]
These are the results of the ITs executed in this [Jenkins build](https://ci.wazuh.info/job/Test_integration/6782/)
[manager_html_report_Test_integration_B6782_20210625084849.zip](https://github.com/wazuh/wazuh-qa/files/6714520/manager_html_report_Test_integration_B6782_20210625084849.zip)

**Comment by jmv74211** [comment]
<details>
  <summary>All tests passed</summary>

```
============================= test session starts ==============================
platform linux -- Python 3.6.9, pytest-6.2.3, py-1.10.0, pluggy-0.13.1 -- /usr/bin/python3
cachedir: .pytest_cache
metadata: {'Python': '3.6.9', 'Platform': 'Linux-4.15.0-1060-aws-x86_64-with-Ubuntu-18.04-bionic', 'Packages': {'pytest': '6.2.3', 'py': '1.10.0', 'pluggy': '0.13.1'}, 'Plugins': {'metadata': '1.8.0', 'testinfra': '5.0.0', 'html': '3.1.1'}}
rootdir: /tmp/Test_integration_B6782_20210625084849/tests/integration, configfile: pytest.ini
plugins: metadata-1.8.0, testinfra-5.0.0, html-3.1.1
collecting ... collected 30 items

test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[agent: Agents' CVEs table: vuln_cves] PASSED [  3%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[agent: Not existing agent] PASSED [  6%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Basics success] PASSED [ 10%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Syntax errors] PASSED [ 13%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Save2 fails] PASSED [ 16%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Integrity_check_global success] PASSED [ 20%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Integrity_check_global fails] PASSED [ 23%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Integrity_check_left success] PASSED [ 26%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Integrity_check_left fails] PASSED [ 30%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Integrity clear success] PASSED [ 33%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Integrity_clear fails] PASSED [ 36%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Invalid agent ID] PASSED [ 40%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Update existing file] PASSED [ 43%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Path length] PASSED [ 46%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Checksum field] PASSED [ 50%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_agent[fim: Large inode] PASSED [ 53%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Insert commands] PASSED [ 56%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Update commands] PASSED [ 60%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Labels commands] PASSED [ 63%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Select commands] PASSED [ 66%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: get-all-agents command] PASSED [ 70%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: sync-agent-info-get command] PASSED [ 73%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: sync-agent-info-set command] PASSED [ 76%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Belongs commands] PASSED [ 80%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Reset connection status] PASSED [ 83%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: get-agents-by-connection-status command] PASSED [ 86%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: disconnect-agents command] PASSED [ 90%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Delete commands] PASSED [ 93%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_messages_global[global: Manager keepalive] PASSED [ 96%]
test_wazuh_db/test_wazuh_db.py::test_wazuh_db_chunks PASSED              [100%]

- generated html file: file:///tmp/Test_integration_B6782_20210625084849/report.html -
============================= 30 passed in 34.80s ==============================
```
</details>

**Review by jmv74211 — APPROVED** [review]

**Pull request merged**